app/assets/stylesheets/pages.scss

```diff
@@ -38,7 +44,7 @@
 
 .home .intro-text p {
   line-height: 1.5;
-  font-size: 1.3rem;
+  font-size: 1rem;
```

Comment:
das gilt ja dann für die gesamte Seite, nicht nur für die Subdomains. Ist das so gewollt? Ansonsten scopen in ooe und vorarlberg

Reply:
`Intro-text` gibt es nur für die subdomains. Meinst du, das sollte man umbenennen?

Reply:
Ich habe `intro-text` jetzt in `intro-subdomain` geändert. OK? So ist es besser verständlich für alle.